package.json

```diff
@@ -66,17 +72,26 @@
     "chalk": "^5.3.0",
     "commander": "^12.1.0",
     "dotenv": "^16.4.5",
-    "eslint": "^8.57.0",
+    "eslint": "^9.8.0",
     "eslint-config-next": "^14.2.5",
```

Comment:
@illionillion 

You can remove `eslint-config-next`, `eslint-config-prettier`.

Reply:
fixed